src/js/setsession.js

```diff
@@ -44,7 +47,7 @@ function startsess() {
     const workInSec = 60 * document.getElementById("pomo_length").value;
     const sbreakInSec = 60 * document.getElementById("sbreak_length").value;
     const lbreakInSec = 60 * document.getElementById("lbreak_length").value;
-    const workPeriods = document.getElementById("work_periods").value;
+    const workPeriods = 1 * document.getElementById("work_periods").value;
```

Comment:
What does multiplying by 1 here do?

If the purpose of this is to change this to a number, this should be more explicit -- for example, using `Number(document.getElementById...)`

Reply:
It was the middle product of trying to fix a bug - it seems something retrieved from local storage is by default a string. Dont need to worry now, all stored numbers are parsed to int when retrieved in script.js. We can discard this.